### ISSUE TITLE
panic with helmfile version v0.61.0

### ISSUE THREAD

**Issue by skaji** [roboll/helmfile#595]
I encountered a panic with  helmfile version v0.61.0.

## Reproduce steps

First, prepare the following helmfile.yaml
```yaml
# helmfile.yaml
repositories:
  - name: incubator
    url: https://kubernetes-charts-incubator.storage.googleapis.com

releases:
  - name: aws-alb-ingress-controller
    chart: incubator/aws-alb-ingress-controller
```

Second, download helmfile version v0.61.0
```
curl -fsSL -o helmfile https://github.com/roboll/helmfile/releases/download/v0.61.0/helmfile_darwin_amd64
chmod +x helmfile
```

Then:
```
❯ ./helmfile template
Adding repo incubator https://kubernetes-charts-incubator.storage.googleapis.com
"incubator" has been added to your repositories

Updating repo
Hang tight while we grab the latest from your chart repositories...
...Skip local chart repository
...Successfully got an update from the "incubator" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈ Happy Helming!⎈

panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x13ae6e2]

goroutine 1 [running]:
github.com/urfave/cli.HandleAction.func1(0xc0001e2e20)
	/go/pkg/mod/github.com/urfave/cli@v0.0.0-20160620154522-6011f165dc28/app.go:474 +0x287
panic(0x145eea0, 0x18737a0)
	/usr/local/go/src/runtime/panic.go:522 +0x1b5
github.com/roboll/helmfile/state.(*ResolvedDependencies).Get(0x0, 0xc0001d215a, 0x1a, 0x9, 0xc0001e1770, 0x1, 0x1)
	/home/circleci/workspace/helmfile/state/chart_dependency.go:109 +0x52
github.com/roboll/helmfile/state.resolveDependencies(0xc0001121c0, 0xc0001e1b58, 0xc00000e0b0, 0xc00000e0b0, 0x0, 0x0)
	/home/circleci/workspace/helmfile/state/chart_dependency.go:175 +0x38b
github.com/roboll/helmfile/state.(*HelmState).mergeLockedDependencies(0xc0001121c0, 0x14cea0d, 0x7, 0xc000092ea0)
	/home/circleci/workspace/helmfile/state/chart_dependency.go:143 +0xe3
github.com/roboll/helmfile/state.(*HelmState).ResolveDeps(...)
	/home/circleci/workspace/helmfile/state/state.go:960
github.com/roboll/helmfile/state.(*HelmState).PrepareReleases(0xc0001121c0, 0x157dde0, 0xc000098540, 0x14cf0c7, 0x8, 0x0, 0x0, 0x0)
	/home/circleci/workspace/helmfile/state/state.go:919 +0x1d1
main.main.func4.1(0xc0001121c0, 0x157dde0, 0xc000098540, 0xc0000803c0, 0x0, 0x1450b00, 0xc0001e1e88)
	/home/circleci/workspace/helmfile/main.go:230 +0x92
github.com/roboll/helmfile/cmd.FindAndIterateOverDesiredStatesUsingFlagsWithReverse.func1(0xc0001121c0, 0x157dde0, 0xc000098540, 0xc0001e2140, 0xc000026600, 0x1a)
	/home/circleci/workspace/helmfile/cmd/cmd.go:74 +0x93
github.com/roboll/helmfile/pkg/app.(*App).VisitDesiredStatesWithReleasesFiltered.func1(0xc0001121c0, 0x157dde0, 0xc000098540, 0x0, 0xc0001b2540, 0x0, 0x100b569)
	/home/circleci/workspace/helmfile/pkg/app/app.go:239 +0x4f5
github.com/roboll/helmfile/pkg/app.(*App).VisitDesiredStates.func1(0x14d0b40, 0xd, 0x14cd56c, 0x1)
	/home/circleci/workspace/helmfile/pkg/app/app.go:196 +0x679
github.com/roboll/helmfile/pkg/app.(*App).visitStateFiles.func1(0x0, 0xc00012c6f0)
	/home/circleci/workspace/helmfile/pkg/app/app.go:104 +0x37
github.com/roboll/helmfile/pkg/app.(*App).within(0xc000160090, 0x14cd56c, 0x1, 0xc00000cbc0, 0xc00012c8e8, 0x2)
	/home/circleci/workspace/helmfile/pkg/app/app.go:51 +0x722
github.com/roboll/helmfile/pkg/app.(*App).visitStateFiles(0xc000160090, 0x0, 0x0, 0xc000066d40, 0x1471260, 0x1883501)
	/home/circleci/workspace/helmfile/pkg/app/app.go:103 +0x283
github.com/roboll/helmfile/pkg/app.(*App).VisitDesiredStates(0xc000160090, 0x0, 0x0, 0x189f328, 0x0, 0x0, 0xc00005c5f0, 0x14b0d01, 0xc00000cba0)
	/home/circleci/workspace/helmfile/pkg/app/app.go:132 +0xcf
github.com/roboll/helmfile/pkg/app.(*App).VisitDesiredStatesWithReleasesFiltered(0xc000160090, 0x0, 0x0, 0xc00000cba0, 0x0, 0x0)
	/home/circleci/workspace/helmfile/pkg/app/app.go:214 +0x98
github.com/roboll/helmfile/cmd.FindAndIterateOverDesiredStatesUsingFlagsWithReverse(0xc00017c140, 0xc00005c500, 0xc00005c5e0, 0x8, 0xc00000cb40)
	/home/circleci/workspace/helmfile/cmd/cmd.go:77 +0xe2
main.findAndIterateOverDesiredStatesUsingFlags(...)
	/home/circleci/workspace/helmfile/main.go:620
main.main.func4(0xc00017c140, 0x0, 0x0)
	/home/circleci/workspace/helmfile/main.go:221 +0x66
reflect.Value.call(0x1442ac0, 0x14eb458, 0x13, 0x14cda48, 0x4, 0xc00012cdc0, 0x1, 0x1, 0xc000066040, 0x1011bc3, ...)
	/usr/local/go/src/reflect/value.go:447 +0x461
reflect.Value.Call(0x1442ac0, 0x14eb458, 0x13, 0xc00012cdc0, 0x1, 0x1, 0xc0000888f0, 0xc000088938, 0x140)
	/usr/local/go/src/reflect/value.go:308 +0xa4
github.com/urfave/cli.HandleAction(0x1442ac0, 0x14eb458, 0xc00017c140, 0x0, 0x0)
	/go/pkg/mod/github.com/urfave/cli@v0.0.0-20160620154522-6011f165dc28/app.go:483 +0x1ff
github.com/urfave/cli.Command.Run(0x14cf0c7, 0x8, 0x0, 0x0, 0x0, 0x0, 0x0, 0x14e3a30, 0x3d, 0x0, ...)
	/go/pkg/mod/github.com/urfave/cli@v0.0.0-20160620154522-6011f165dc28/command.go:186 +0x8d1
github.com/urfave/cli.(*App).Run(0xc000001080, 0xc00009a000, 0x2, 0x2, 0x0, 0x0)
	/go/pkg/mod/github.com/urfave/cli@v0.0.0-20160620154522-6011f165dc28/app.go:237 +0x601
main.main()
	/home/circleci/workspace/helmfile/main.go:579 +0x20ba
```

**Comment by mumoshu** [comment]
@skaji I was able to fix it in #596 thanks to your detailed report ☺️

**Comment by skaji** [comment]
@mumoshu Thanks!

**Comment by mumoshu** [comment]
Released v0.61.1 for this. Please let me know if the issue persisted somehow. Thanks!
